docs/DeveloperGuide.md

```diff
@@ -573,12 +589,11 @@ The following section describes the challenges our team faced in this project, t
 
 #### Extension of Model
 
-{Zech to elaborate on the challenges faced when extending the model
-some highlights:
-added new fields
-a great deal of refactoring was done
-model was reworked to contain a passenger and pool list
-the initial person class was refactored to become driver and passenger}
+As our group had initially decided on morphing the existing AB3, changes to the model consisted mainly of refactoring existing classes. Although a rather trivial task, this process was rather time-consuming and tedious. Due to the scale of the existing AB3 code, some difficulty was encountered during this refactoring stage as some making seemingly minute changes in method signatures or names would result in a need to edit other segments of affected code. Although the use of IntelliJ’s toolbox greatly sped up the process, the nature of some of the refactoring changes we intended to make meant that some of said changes were not as easily detected by IntelliJ, resulting in some uncaught bugs following our refactoring stage. The brownfield nature of the project also lead to some difficulty in understanding the model and how the different modules interact with each other.
```

Comment:
Maybe 

The brownfield nature of the project also _led_ to some...

Reply:
Right my mistake

Reply:
i think you could omit the "although a trivial task" beacause the idea is to show that it is not trivial